apps/dashboard/src/routes/account/+page.svelte

```diff
@@ -77,19 +54,40 @@
 		name="website"
 		type="url"
 		class="input mb-4"
-		value={form?.website ?? website}
+		bind:value={website}
 	/>
 	<div>
-		<input
-			type="submit"
-			form="account-info"
+		<button
+			on:click={async () => {
+				const email = session.user.email;
+				// todo display error
+				if (!email) return;
+
+				// todo display error
+				const { error } = await data.supabase.from('profiles').upsert({
```

Comment:
We can probably use a notification to showcase a failure and or successful update.